src/ServicePulse.Host/app/js/services/services.platform-update.js

```diff
@@ -2,30 +2,41 @@
 (function(window, angular, undefined) {
     "use strict";
 
-    function service($http, scConfig) {
-
-        function _getReleases() {
-            return $http
-                .get(scConfig.service_pulse_url, {
-                     responseType: "json"
-                })
-                .then(function(response) {
-                    return response;
-                }, function(response) {
-                    return {
-                        // suppress error
-                        data: []
-                    };
+    function service($http, $q) {
+
+        function getReleases() {
+
+
+            var serviceProductUrls = [
+                { product: 'SP', url: '//platformupdate.particular.net/servicepulse.txt' },
+                { product: 'SC', url: '//platformupdate.particular.net/servicecontrol.txt' }
+            ];
+
+            return $q.all(serviceProductUrls.map(function(item) {
+                    return $http({
+                        method: 'GET',
+                        url: item.url
+                    });
+                }))
+                .then(function(results) {
+                    var resultObj = {};
+                    results.forEach(function(val, i) {
+                        resultObj[serviceProductUrls[i].product] = val.data;
+                    });
+                    return resultObj;
+                }, function () {
```

Comment:
Shouldn't swallow the error here.

We're returning a promise, so let the caller of `getReleases()` decide whether to swallow the exception or not.

I would just leave the fail overload out so that callers can decide what to do.